source/content/migrate-wordpress-site-networks.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
Can/should we edit this file name (which I assume is in turn the URL path) itself and add a redirect, or leave as-is?

Reply:
We can update the file name and create a redirect. Is that the preferred option here?

Reply:
@scottbuscemi should be able to answer that 😄

Reply:
I agree that if we can rename and redirect it would be ideal.